modules/core/02-client/types/msgs_test.go

```diff
@@ -678,3 +680,87 @@ func TestMsgUpdateParamsGetSigners(t *testing.T) {
 		}
 	}
 }
+
+// TestMsgRecoverClientValidateBasic tests ValidateBasic for MsgRecoverClient
+func (suite *TypesTestSuite) TestMsgRecoverClientValidateBasic() {
+	var msg *types.MsgRecoverClient
+
+	signer := ibctesting.TestAccAddress
```

Comment:
I think this var can be removed, right?

Reply:
can be moved inline during struct initialization, yea